src/client/driver/storage.js

```diff
@@ -24,16 +24,23 @@ export default class Storage {
     }
 
     _loadFromStorage () {
-        const savedData = nativeMethods.storageGetItem.call(this.storage, this.storageKey);
+        const savedData = nativeMethods.storageGetItem.call(this.storage, this.storageKey)
+            || window['%proxylessContextStorage%'];
 
         if (savedData) {
             this.data = JSON.parse(savedData);
+
+            window['%proxylessContextStorage%'] = null;
```

Comment:
I propose move out it to the separate entity since it doesn't connected with session  storage.

Reply:
Also, define window properties in safe manner - https://github.com/DevExpress/testcafe/blob/master/src/client/core/index.js#L85

Reply:
we do not have nativeMethods at this moment, so I've just used Object.defineProperty.